notion-docs/scripts/update.ts

```diff
@@ -242,7 +242,7 @@ const getContentFromCMS = async (): Promise<CMSContents> => {
 // Renderer for FAQs structured data in JSON
 const renderJSONFAQStructuredData = (faqs: RenderedKnowledgeItem[]) => {
   const printItem = (faq: RenderedKnowledgeItem) => {
-    const faqQuestion = escapeForJSON(faq.title)
+    const faqQuestion = escapeForJSON(faq.titleforSort)
```

Comment:
[titleForSort](https://github.com/OffchainLabs/notion-docs-generator/blob/main/src/item.ts#L39) strips all HTML and Markdown characters. Titles don't use any of this, but this change should prevent potential future issues with unexpected characters/formatting.